akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -60,6 +77,73 @@ class GracefulTerminationSpec
       ex.getMessage should include("Connection refused")
     }
 
+    "fail chunked response streams" in new TestSetup {
+      val clientSystem = ActorSystem("client")
+      val r1 =
+        Http()(clientSystem).singleRequest(nextRequest, connectionContext = clientConnectionContext, settings = basePoolSettings)
+
+      // reply with an infinite entity stream
+      val chunks = Source
+        .fromIterator(() => Iterator.from(1).map(v => ChunkStreamPart(s"reply$v,")))
+        .throttle(1, 300.millis)
+      reply(_ => HttpResponse(entity = HttpEntity.Chunked(ContentTypes.`text/plain(UTF-8)`, chunks)))
+
+      // start reading the response
+      val response = r1.futureValue.entity.dataBytes
+        .via(Framing.delimiter(ByteString(","), 20))
+        .runWith(TestSink.probe[ByteString])
+      response.requestNext().utf8String should ===("reply1")
+
+      try {
+        val termination = serverBinding.terminate(hardDeadline = 50.millis)
+        response.request(20)
+        // local testing shows the stream fails long after the 50 ms deadline
+        response.expectNext().utf8String should ===("reply2")
+        response.expectNext().utf8String should ===("reply3")
```

Comment:
Should we remove a couple of these as well?

Reply:
Done.